Source/WebCore/Modules/model-element/HTMLModelElement.h

```diff
@@ -163,8 +163,10 @@ class HTMLModelElement final : public HTMLElement, private CachedRawResourceClie
     void setCurrentTime(double);
 #endif
 
-#if ENABLE(MODEL_ELEMENT_STAGE_MODE_INTERACTION)
+#if ENABLE(MODEL_ELEMENT_STAGE_MODE)
     WEBCORE_EXPORT bool supportsStageModeInteraction() const;
```

Comment:
Seems pretty weird for a function with the text "StageModeInteraction" to not be in the ENABLE(MODEL_ELEMENT_STAGE_MODE_INTERACTION) block.